platform/viewer/public/config/local_static.js

```diff
@@ -12,9 +13,9 @@ window.config = {
       sourceName: 'dicomweb',
       configuration: {
         name: 'DCM4CHEE',
-        wadoUriRoot: '/dicomweb',
-        qidoRoot: '/dicomweb',
-        wadoRoot: '/dicomweb',
+        wadoUriRoot: 'http://localhost:5000/dicomweb',
```

Comment:
Need to use a different port for testing the local static server, as doing it through deploy is quite slow.